Gemfile

```diff
@@ -0,0 +1,35 @@
+source 'https://rubygems.org'
+
+gem 'rails', '~> 5.0'
+# Use sqlite3 as the database for Active Record
```

Comment:
Drop all comments, if its not documentation. There should be postgres (in all env)